Sources/Models/API/Product.swift

```diff
@@ -94,6 +94,8 @@ struct Product: Mappable {
     var nutriscore: String?
     private var nutriscoreWarningNoFruitsVegetablesNutsAsInt: Int?
     private var nutriscoreWarningNoFiberAsInt: Int?
+    var attributeGroup: AttributeGroup?
+
```

Comment:
I believe that since attributes_group belongs to a product in the returned JSON, then attributeGroup should be a property of `product` struct

Reply:
I changed this to an array `[AttributeGroup]?` and it's working now. All Attribute Groups and their Attributes are properly mapped.
The product I tested with is [creme-entiere-fluide-bio-30-mg-auchan](creme-entiere-fluide-bio-30-mg-auchan)

![mapping_attribute_groups_debug_success](https://user-images.githubusercontent.com/25991237/95643617-de2eea00-0a75-11eb-9047-a718ebf4df32.png)